docs/mp/lra.adoc

```diff
@@ -0,0 +1,461 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2021, 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= LRA
+:toc:
+:toc-placement: preamble
+:description: Long Running Actions
+:keywords: helidon, mp, lra
+:h1Prefix: MP
+:feature-name: Long Running Actions
+:spec-version: 1.0-RC3
+:spec-name: Micro Profile {feature-name} specification
+:spec-link: https://download.eclipse.org/microprofile/microprofile-lra-{spec-version}/microprofile-lra-spec-{spec-version}.html
```

Comment:
A variable holding the spec URL is already defined in attributes.adoc. It doesn't make sense to redefine it here.

Reply:
applied